MainScreenStackWidget.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
To keep things nice and clean in the working directory, we should put all the graphical stuff in the `views` directory.